taskcluster/ci/toolchain/conda.yml

```diff
@@ -5,64 +5,24 @@
 task-defaults:
     description: "Conda Pack an enviroment"
```

Comment:
Should conda.yml be renamed to conda_windows.yml to match the naming pattern of conda_android.yml? Consider if the description here should say 'windows environment', and if the description in conda_android.yml should be similarly updated. (Nit: typo in 'environment'.)

Reply:
Fixed the typo, and moved that to a `conda_windows.yml` good callout thanks!